lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -655,6 +658,43 @@ def test_coach_tab_for_ccx_advance_settings(self, ccx_feature_flag, enable_ccx,
             )
 
 
+class TestStudentDashboardWithCCX(ModuleStoreTestCase):
+    """
+    Test to ensure that the student dashboard works for users enrolled in CCX
+    courses.
+    """
+
+    def setUp(self):
+        """
+        Set up courses and enrollments.
+        """
+        super(TestStudentDashboardWithCCX, self).setUp()
+
+        # Create a Draft Mongo and a Split Mongo course and enroll a student user in them.
+        self.student_password = "foobar"
+        self.student = UserFactory.create(username="test", password=self.student_password, is_staff=False)
+        self.draft_course = CourseFactory.create(default_store=ModuleStoreEnum.Type.mongo)
+        self.split_course = CourseFactory.create(default_store=ModuleStoreEnum.Type.split)
+        CourseEnrollment.enroll(self.student, self.draft_course.id)
+        CourseEnrollment.enroll(self.student, self.split_course.id)
```

Comment:
@cpennington I don't have a good answer to that, but I don't think changing it is worth delaying the hotfix at this point.